mw-webapp/src/service/CurrentProblemService.ts

```diff
@@ -1,16 +1,22 @@
-import {get, ref} from "firebase/database";
+import {collection, getDocs} from "firebase/firestore";
 import {CurrentProblemDTOToCurrentProblemConverter} from "src/converter/CurrentProblemConverter";
 import {db} from "src/firebase";
 import {CurrentProblem} from "src/model/businessModel/CurrentProblem";
 import {CurrentProblem as CurrentProblemDTO} from "src/model/firebaseCollection/CurrentProblem";
 
+const PATH_TO_CURRENT_PROBLEMS_COLLECTION = "currentProblems";
+
 export class CurrentProblemService {
 
-  public static async onValueFromRealTimeDb(): Promise<CurrentProblem[]> {
-    const snapshot = await get(ref(db, "/currentProblems"));
-    const currentProblemsRaw: CurrentProblemDTO[] = await snapshot.val();
-    const currentProblems: CurrentProblem[] = currentProblemsRaw.map((item) =>
-      CurrentProblemDTOToCurrentProblemConverter(item));
+  public static async getCurrentProblems(): Promise<CurrentProblem[]> {
+    const currentProblemsRaw = await getDocs(collection(db, PATH_TO_CURRENT_PROBLEMS_COLLECTION));
+    const currentProblemsDTO: CurrentProblemDTO[] = currentProblemsRaw.docs.map((currentProblemRaw) => ({
+      uuid: currentProblemRaw.data().uuid,
+      description: currentProblemRaw.data().description,
+      isDone: currentProblemRaw.data().isDone,
+    }));
+    const currentProblems = currentProblemsDTO
+      .map((currentProblemDTO) => CurrentProblemDTOToCurrentProblemConverter(currentProblemDTO));
```

Comment:
1. use short syntax:
`...((currentProblemDTO) => CurrentProblemDTOToCurrentProblemConverter(currentProblemDTO))`
the same:
`...(CurrentProblemDTOToCurrentProblemConverter)

2. Why converter starts with Upper letter

Reply:
Done